app/assets/stylesheets/articles.scss

```diff
@@ -944,10 +948,12 @@
               display: block;
               &:hover {
                 background: $light-purple;
+                background: var(--theme-container-background, $light-purple);
```

Comment:
--theme-container-background-hover is now a thing which may be better suited here.